docs/cron-workflows.md

```diff
@@ -2,10 +2,95 @@
 
 ![alpha](assets/alpha.svg)
 
-> v2.5 and after
+Available on `v2.5.0`+
```

Comment:
um, this formatting is used in other docs, I don't think we should change it

Reply:
Sure thing